### DISCUSSION TITLE
NASA - Send custom value to specific message number?

### DISCUSSION THREAD

**Discussion by Oggy123** [omerfaruk-aran/esphome_samsung_hvac_bus#115] [General]
Hello gentlemen, 

first, I want to say huge thank you for this project! Till now I send IR codes via ESP8266 IR modules in each indoor unit (I have four ar07txc...), but I missed feedback. Now I'm able to control and read current state, it's awesome. 
I have home automation build on node-red actions over MQTT without HA, but it's not much important. 

I have set power saving mode on outdoor unit, it stops communication over F1F2 after 5 minutes from last heating/cooling and waits for wakeup by indoor unit. It is realized by sending 0 to 0x0023. 

Is it possible to send some command over MQTT to send custom value to any message address to simulate wakeup from indoor unit (or any other value to specified message address)?  

And off course, if you need someone to test anything on NASA, don't hesitate to let me know

Many thanks!

**Comment by lanwin** [comment]
Hi @Oggy123 can you tell my a little bit more on how the power saving mode works? My unit did not seem to have that. How did you activate it and what it is for?

**Reply by Oggy123** [reply]
Hi @lanwin, you can find some more info here https://www.manualslib.com/manual/2404337/Samsung-Aj-Series.html?page=36#manual 
You can activate it only on some models and only on outdoor unit. Samsung promises it can save few more watts on standby, but the bus is completely muted, no message is transferred (even from indoor units) until some indoor device wakes outdoor unit via mentioned message. 

Attached you can find log with short description 
[power save mode.txt](https://github.com/lanwin/esphome_samsung_ac/files/14961637/power.save.mode.txt)
